library/globals.inc.php

```diff
@@ -1866,7 +1866,12 @@
       '0',
       xl('Enable Onsite Patient Portal.')
     ),
-
+    'portal_statements' => array(
```

Comment:
This feature is in this branch reviewed by Art
https://github.com/juggernautsei/openemr/tree/portal_invoice

Thread is here
[Patient Portal Statements](https://sourceforge.net/p/openemr/discussion/202506/thread/ce222227/?limit=25#8200)